services/distribution/src/main/resources/application.yaml

```diff
@@ -84,9 +84,6 @@ spring:
   main:
     web-application-type: NONE
   # Postgres configuration
-  jpa:
-    hibernate:
-      ddl-auto: validate
```

Comment:
Does some other component perform a schema validation now?

Reply:
To some extent you could say that Flyway is ensuring the presence of the expected schema at startup. Beyond that we will have to, for example, rely on unit tests to detect bugs.